File: features/recharge/digitalproduct/src/main/java/com/tokopedia/digital/newcart/domain/ICheckoutRepository.java
package com.tokopedia.digital.newcart.domain;

import com.tokopedia.digital.newcart.data.entity.requestbody.checkout.RequestBodyCheckout;
import com.tokopedia.digital.newcart.domain.model.CheckoutDigitalData;

import rx.Observable;

/**
 * @author anggaprasetiyo on 3/9/17.
 */

public interface ICheckoutRepository {

    Observable<CheckoutDigitalData> checkoutCart(RequestBodyCheckout requestBodyCheckout);

}

File: features/transaction/checkout/src/main/java/com/tokopedia/checkout/data/repository/ICheckoutRepository.kt
package com.tokopedia.checkout.data.repository

import com.tokopedia.checkout.data.model.response.saveshipmentstate.SaveShipmentStateResponse
import rx.Observable

/**
 * Created by Irfan Khoirul on 2019-08-15.
 */

interface ICheckoutRepository {

    fun saveShipmentState(params: Map<String, String>): Observable<SaveShipmentStateResponse>

}